### ISSUE TITLE
Document setup of SecureDrop developer environment in Qubes OS

### ISSUE THREAD

**Issue by eloquence** [freedomofpress/securedrop-workstation#106]
To enable developers to do much of their work directly on Qubes, let's add documentation for setting up the Docker-based dev environment for SecureDrop.

**Comment by conorsch** [comment]
Working actively on #105 (staging docs), will try to keep these changes in a separate PR to aid in the review, but they may wind up together in the interest of clarity: the dev env includes a lot of reqs for staging, as well.

**Comment by conorsch** [comment]
After some discussion with @joshuathayer, we realized we were interpreting the scope of this ticket differently. What we've now decided is:

1. _This_ ticket tracks adding documentation to the SecureDrop repository explaining how to run the container-based dev env on a Qubes workstation. 
2. A _separate_ ticket should track how to work on the SecureDrop Workstation code on a Qubes workstation, with particular attention to supporting the forthcoming SD API. 

Note that 2) may require dev env changes in one or several repositories. 1) should be docs-only.
